docker/start_s3rver.sh

```diff
@@ -13,8 +13,8 @@ if lsof -i :5000 > /dev/null ; then
     exit
 fi
 
-mkdir -p /s3rver
-node_modules/.bin/s3rver --directory /s3rver --port 5000 --configure-bucket workspaces --configure-bucket chunks --configure-bucket file-store > /dev/null &
+mkdir -p ./s3rver
+node_modules/.bin/s3rver --directory ./s3rver --port 5000 --configure-bucket workspaces --configure-bucket chunks --configure-bucket file-store > /dev/null &
```

Comment:
I tried to run this with `make start` on my machine outside of docker, but the root of my machine (`/`) is read-only, so this failed. I opted to change this to create the directory in PrairieLearn instead. See the corresponding change to `.gitignore`.

Reply:
This seems fine. We were keeping in `/` assuming everyone was running inside a container, but I'm on board with supporting non-containerized dev.